sql/core.go

```diff
@@ -407,6 +439,10 @@ type StatisticsTable interface {
 	NumRows(*Context) (uint64, error)
 	// DataLength returns the length of the data file (varies by engine).
 	DataLength(ctx *Context) (uint64, error)
+	// CalculateStatistics fills in the histogram object for the associate table.
+	CalculateStatistics(ctx *Context) error
+	// GetStatistics returns the statistics object for the associated table,
```

Comment:
GetStatistics returns statistics for this table

Reply:
Just call this `Statistics()`, that's the pattern we use for most of these accessor interface (no `Get`)